setup.cfg

```diff
@@ -33,8 +33,8 @@ packages = find_namespace:
 include_package_data = true
 install_requires =
     apache-airflow>=2.1
-    great-expectations>=0.13.14
-    sqlalchemy>=1.3.16
+    great-expectations==0.15.34
+    sqlalchemy==1.4.44
```

Comment:
```suggestion
    great-expectations>=0.15.34
    sqlalchemy>=1.3.16
```

Reply:
We should not pin the version -- otherwise it won't work with other libraries. Please keep the upper bound open